src/graphql/device/TypeDefs.js

```diff
@@ -0,0 +1,91 @@
+const TypeDefs = [`
+    enum Type{
+        dynamic
+        static
+        actuate
+    }
+    enum ValueType {
+        integer
```

Comment:
Enums should be uppercase?

Reply:
``
enum ValueType {
   NUMBER
   STRING
   BOOLEAN
   GEO
   UNDEFINED
}
` `